app/models/post.rb

```diff
@@ -5,6 +5,10 @@ class Post < ApplicationRecord
 
   after_save :update_post_counter
 
+  def all_comments
+    comments.includes([:author]).order(created_at: :desc)
+  end
```

Comment:
- [ ] Kindly make sure to solve the N+1 problem when fetching all posts and their comments, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Make%20sure%20that%20the%20N%2B1%20problem%20is%20solved%20when%20fetching%20all%20posts%20and%20their%20comments%20for%20a%20user) requirement. This can be done in the `index` action in the `Posts` controller

Reply:
Done. Thanks for the review! @ShadyShawkat